packages/wrangler/src/api/startDevWorker/types.ts

```diff
@@ -184,6 +186,7 @@ export type StartDevWorkerOptions = StartDevWorkerInput & {
 		persist: string;
 	};
 	entrypoint: string;
+	experimentalAssets?: ExperimentalAssets;
```

Comment:
This will already be on this type via the `StartDevWorkerInput &` intersection

Reply:
✅